cli/cmd/main.go

```diff
@@ -283,6 +284,11 @@ func main() {
 						Required: false,
 						Usage:    "mime_types filter",
 					},
+					&cli.IntFlag{
+						Name:     dataFlagConcurrentDownloads,
+						Required: false,
+						Usage:    "number of download requests to make in parallel",
```

Comment:
Would make a note to explicitly set this value to 1 for consecutive downloads, since we otherwise default to 10 concurrent requests.

Reply:
Added not about default value but left out the note about needing to set to 1 for non-parallel downloads, because I think it's directly implied